_posts/2022-04-11-changelog-168.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #168"
+date: 2022-04-11 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#8647] Collapse macro expansion steps into single one.
+It's not only fixes UI blinking during macro expansion process but also significantly speed up macro expansion itself
+
+* [#8689] Provide file links in terminal for `dbg!` macro output
+
+  {% include gif-img.html path="/assets/posts/changelog-168/dbg_output_file_links" w="700px" %}
+
+## Fixes
+
+* [#8674] The `Generate constructor` action now places the cursor after the generated constructor (by [@Kobzol])
+
+* [#8671] Escape keyword-like identifiers like `type` by `Import` quick-fix
+
+* [#8631] Properly evaluate build scripts even if a crate doesn't contain binary or library targets.
+  Note, build script evaluation is disabled by default. To turn it on, enable `org.rust.cargo.evaluate.build.scripts` [experimental feature][experimental features]
+
+* [#8692] Update [ra_ap_proc_macro_srv](https://crates.io/crates/ra_ap_proc_macro_srv) lib to 0.0.102 in procedural macro expander.
+  It should fix expansion of some procedural macro calls (for example, from [rocket](https://crates.io/crates/rocket) crate).
+  Note, procedural macro expansion is disabled by default for now.
```

Comment:
We already used "Note, build script evaluation is disabled by default" before, so I suggest either removing "for now" part here or adding it to the first one.